src/test/java/org/opentripplanner/gtfs/mapping/StopTimeMapperTest.java

```diff
@@ -85,36 +96,60 @@ public class StopTimesMapperTest {
     new TranslationHelper()
   );
 
-  static {
+  /**
+   * Build a static ("regular") stop.
+   */
+  private static Stop buildStop() {
+    var stop = new Stop();
+    stop.setId(AGENCY_AND_ID);
+    stop.setName(STOP_NAME);
+    stop.setLat(53.12);
+    stop.setLon(12.34);
+    return stop;
+  }
+
+  /**
+   * Builds a stop time with a fixed stop.
+   */
+  static StopTime buildDefaultStopTime() {
+    var stopTime = buildStopTime();
+    var stop = buildStop();
+    stopTime.setStop(stop);
+    return stopTime;
+  }
+
+  /**
+   * Builds a stop time without a stop. Useful for testing flex fields.
+   */
+  private static StopTime buildStopTime() {
     TRIP.setId(AGENCY_AND_ID);
-    STOP.setId(AGENCY_AND_ID);
-    STOP.setName(STOP_NAME);
-
-    STOP_TIME.setId(ID);
-    STOP_TIME.setArrivalTime(ARRIVAL_TIME);
-    STOP_TIME.setDepartureTime(DEPARTURE_TIME);
-    STOP_TIME.setDropOffType(DROP_OFF_TYPE);
-    STOP_TIME.setFarePeriodId(FARE_PERIOD_ID);
-    STOP_TIME.setPickupType(PICKUP_TYPE);
-    STOP_TIME.setRouteShortName(ROUTE_SHORT_NAME);
-    STOP_TIME.setShapeDistTraveled(SHAPE_DIST_TRAVELED);
-    STOP_TIME.setStop(STOP);
-    STOP_TIME.setStopHeadsign(HEAD_SIGN);
-    STOP_TIME.setStopSequence(STOP_SEQUENCE);
-    STOP_TIME.setTimepoint(TIMEPOINT);
-    STOP_TIME.setTrip(TRIP);
+
+    var stopTime = new StopTime();
+    stopTime.setId(ID);
+    stopTime.setArrivalTime(ARRIVAL_TIME);
+    stopTime.setDepartureTime(DEPARTURE_TIME);
+    stopTime.setDropOffType(DROP_OFF_TYPE);
+    stopTime.setFarePeriodId(FARE_PERIOD_ID);
+    stopTime.setPickupType(PICKUP_TYPE);
+    stopTime.setRouteShortName(ROUTE_SHORT_NAME);
+    stopTime.setShapeDistTraveled(SHAPE_DIST_TRAVELED);
+    stopTime.setStopHeadsign(HEAD_SIGN);
+    stopTime.setStopSequence(STOP_SEQUENCE);
+    stopTime.setTimepoint(TIMEPOINT);
+    stopTime.setTrip(TRIP);
+    return stopTime;
   }
 
   @Test
   public void testMapCollection() {
     assertNull(subject.map((Collection<StopTime>) null));
     assertTrue(subject.map(Collections.emptyList()).isEmpty());
-    assertEquals(1, subject.map(Collections.singleton(STOP_TIME)).size());
+    assertEquals(1, subject.map(Collections.singleton(buildDefaultStopTime())).size());
   }
 
   @Test
   public void testMap() {
-    org.opentripplanner.model.StopTime result = subject.map(STOP_TIME);
+    org.opentripplanner.model.StopTime result = subject.map(buildDefaultStopTime());
```

Comment:
We do we use this long type specification in some places in this file? Maybe just use var?

Reply:
Good point. Fixed in https://github.com/opentripplanner/OpenTripPlanner/pull/5564/commits/eeab6116f735573fc5421ace88aa3ff10e6c8e86